vital/types/metadata.h

```diff
@@ -39,24 +39,24 @@
 #include <vital/vital_export.h>
 
 #include <vital/any.h>
-
-#include <vital/types/timestamp.h>
 #include <vital/exceptions/metadata.h>
 #include <vital/types/metadata_tags.h>
+#include <vital/types/timestamp.h>
```

Comment:
Heh, for #1012, I figured I was changing enough and tried to avoid style changes (except for `any.h`); +1 for fixing it!